web/src/user/LibraryApplication/index.ts

```diff
@@ -62,23 +64,44 @@ export class LibraryApplication extends AKElement {
         ];
     }
 
+    renderExpansion(application: Application) {
```

Comment:
Consider "view file" using the three-dot menu to your upper-right.  It'll make more clear what's changed.